### ISSUE TITLE
Ubuntu1404 failing to build at vmtools phase.  Kernel Panic Looks like.

### ISSUE THREAD

**Issue by derhornspieler** [rapid7/metasploitable3#318]
Description
Failing to Build Ubuntu1404

Host System

OS: macOS High Sierra 10.13.6 (17G65)
Packer Version: 1.2.5
Vagrant Version: 2.1.2
VirtualBox Version: 5.2.14 r123301 (Qt5.6.3)

## Command Output
==> virtualbox-iso: Provisioning with chef-solo
    virtualbox-iso: Installing Chef...
    virtualbox-iso:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
    virtualbox-iso:                                  Dload  Upload   Total   Spent    Left  Speed
    virtualbox-iso: 100 23075  100 23075    0     0  31654      0 --:--:-- --:--:-- --:--:-- 31652
    virtualbox-iso: ubuntu 14.04 x86_64
    virtualbox-iso: Getting information for chef stable 13.8.5 for ubuntu...
    virtualbox-iso: downloading https://omnitruck.chef.io/stable/chef/metadata?v=13.8.5&p=ubuntu&pv=14.04&m=x86_64
    virtualbox-iso:   to file /tmp/install.sh.999/metadata.txt
    virtualbox-iso: trying wget...
    virtualbox-iso: sha1	1ecb70effcf557dbac0d0de528bbef6937f9c581
    virtualbox-iso: sha256	ce0ff3baf39c8c13ed474104928e7e4568a4997a1d5797cae2b2ba3ee001e3a8
    virtualbox-iso: url	https://packages.chef.io/files/stable/chef/13.8.5/ubuntu/14.04/chef_13.8.5-1_amd64.deb
    virtualbox-iso: version	13.8.5
    virtualbox-iso: downloaded metadata file looks valid...
    virtualbox-iso: downloading https://packages.chef.io/files/stable/chef/13.8.5/ubuntu/14.04/chef_13.8.5-1_amd64.deb
    virtualbox-iso:   to file /tmp/install.sh.999/chef_13.8.5-1_amd64.deb
    virtualbox-iso: trying wget...
    virtualbox-iso: Comparing checksum with sha256sum...
    virtualbox-iso: Installing chef 13.8.5
    virtualbox-iso: installing with dpkg...
    virtualbox-iso: Selecting previously unselected package chef.
    virtualbox-iso: (Reading database ... 61946 files and directories currently installed.)
    virtualbox-iso: Preparing to unpack .../chef_13.8.5-1_amd64.deb ...
    virtualbox-iso: Unpacking chef (13.8.5-1) ...
    virtualbox-iso: Setting up chef (13.8.5-1) ...
    virtualbox-iso: Thank you for installing Chef!
    virtualbox-iso: Creating directory: /tmp/packer-chef-solo
    virtualbox-iso: Creating directory: /tmp/packer-chef-solo/cookbooks-0
    virtualbox-iso: Creating configuration file 'solo.rb'
    virtualbox-iso: Creating JSON attribute file
    virtualbox-iso: Executing Chef: sudo chef-solo --no-color -c /tmp/packer-chef-solo/solo.rb -j /tmp/packer-chef-solo/node.json
    virtualbox-iso: Starting Chef Client, version 13.8.5
    virtualbox-iso: resolving cookbooks for run list: ["metasploitable::vm_tools", "metasploitable::users", "metasploitable::mysql", "metasploitable::apache_continuum", "metasploitable::apache", "metasploitable::php_545", "metasploitable::phpmyadmin", "metasploitable::proftpd", "metasploitable::docker", "metasploitable::samba", "metasploitable::sinatra", "metasploitable::unrealircd", "metasploitable::chatbot", "metasploitable::payroll_app", "metasploitable::readme_app", "metasploitable::cups", "metasploitable::drupal", "metasploitable::knockd", "metasploitable::iptables", "metasploitable::flags"]
    virtualbox-iso: Synchronizing Cookbooks:
    virtualbox-iso:   - metasploitable (0.1.0)
    virtualbox-iso:   - docker (2.15.2)
    virtualbox-iso:   - mysql (8.2.0)
    virtualbox-iso: Installing Cookbook Gems:
    virtualbox-iso: Compiling Cookbooks...
    virtualbox-iso: Converging 174 resources
    virtualbox-iso: Recipe: metasploitable::vm_tools

<img width="650" alt="screen shot 2018-07-18 at 1 56 31 pm" src="https://user-images.githubusercontent.com/15236687/42898880-6d6cf2b8-8a92-11e8-90b2-ca0376b102d0.png">
Failing to complete build

**Comment by lsoumille** [comment]
same issue here too

**Comment by 3ndG4me** [comment]
Also can confirm, having the same issue.

**Comment by asoto-r7** [comment]
`/var/log` points to an kernel driver issue with `vboxvideo`:

```
vboxguest: Successfully loaded version 5.2.16 (interface 0x00010004)
vboxsf: Successfully loaded version 5.2.16 (interface 0x00010004)
[drm] Initialized drm 1.1.0 20060810
BUG: unable to handle kernel NULL pointer dereference at 0000000000000030
IP: [<ffffffffa02cec6e>] drm_dev_register+0x2e/0x1e0 [drm]
PGD 0 
Oops: 0000 [#1] SMP 
Modules linked in: vboxvideo(OF+) [ ... ]
```

There's a bug in VirtualBox guest tools that tries to use a kernel function that isn't present in our older kernel ([source](https://www.virtualbox.org/pipermail/vbox-dev/2017-June/014477.html)).

@bcook is taking a deeper look, but as a workaround, edit the `packer/templates/ubuntu_1404.json` file and remove line 95:

```
             "metasploitable3::vm_tools"
```

**Comment by busterb** [comment]
Fix in #321. While I found that a newer release stopped panics from happening, I don't see any reason to actually include vm-tools anyway. So I discarded it and used an even more vulnerable base kernel image instead.
